tests/test_models.py

```diff
@@ -50,6 +50,7 @@ def tearDown(self):
         """ This runs after each test """
         db.session.remove()
         db.drop_all()
+        db.get_engine().dispose()
```

Comment:
wondering if we should also be doing this in test_routes.py

Reply:
oh sure

Reply:
> 

Done.